stingray/lightcurve.py

```diff
@@ -688,3 +688,17 @@ def read(self, filename, format_='pickle'):
 
         else:
             utils.simon("Format not understood.")
+
+    def split_by_gti(self):
+        """Splits the `LightCurve` into a list of `LightCurve`s , using GTIs."""
```

Comment:
Could this method conceivably take an optional keyword "gti", in case the user wants to use different GTIs to those stored in the `Lightcurve` method or do we just not expect that to happen?